backend/app/services/share/team_share_service.py

```
@@ -81,6 +105,212 @@ def _get_resource_owner_id(self, resource: Kind) -> int:
+        current_user_id: int,
+        members_data: List[
+            Tuple[int, BaseRole, Optional[str], Optional[str], Optional[str]]
+        ],
+    ) -> BatchResourceMemberResponse:
+        """Batch add Team members and filter invalid child namespace grants."""
+        self._ensure_can_manage_members(db, resource_id, current_user_id)
+
+        filtered_members = []
+        failed: List[FailedMemberResponse] = []
+        for entry in members_data:
+            target_user_id, role, entity_type, entity_id, entity_display_name = entry
```

Ruff (0.15.17)

[warning] Unpacked variable `entity_display_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)